### ISSUE TITLE
I use laravel 7 error message cannot read directory

### ISSUE THREAD

**Issue by huylong** [ckfinder/ckfinder-laravel-package#42]
I use laravel 7 when install success package ckfinder. I think it just happens version laravel 7. PLZ help me fix 
Url: http://localhost:8081/ckfinder/connector?command=Init&lang=en&type=Images
Error: 
```
TypeError
Argument 1 passed to CKSource\CKFinder\CKFinder::handleOptionsRequest() must be an instance of Symfony\Component\HttpKernel\Event\GetResponseEvent, instance of Symfony\Component\HttpKernel\Event\RequestEvent given, called in /var/www/html/blog/vendor/symfony/event-dispatcher/EventDispatcher.php on line 230,```

**Comment by nguyendai0409** [comment]
The same issue

**Comment by truoggiag** [comment]
![image](https://user-images.githubusercontent.com/38307035/77821574-fc5a7b80-711d-11ea-8ec8-013b2abc06cc.png)
The same issue

**Comment by truoggiag** [comment]
![image](https://user-images.githubusercontent.com/38307035/77821597-1eec9480-711e-11ea-869b-ff847db75e2a.png)
The same issue

**Comment by bryanjack** [comment]
Same issue

**Comment by holfbroy** [comment]
![image](https://user-images.githubusercontent.com/3446208/78345893-ef6adb80-75a6-11ea-80f4-8d58fa7f2706.png)
same issue, 
please, consider this matter as urgent, thank you

**Comment by Cang16** [comment]
I'm using laravel 7 and has same issue
Does anyone solve this?

**Comment by congngokmt** [comment]
I was upgrade 6.x to 7.x with same issue

**Comment by sunardo** [comment]
Does anyone solve this issue?

**Comment by johenel** [comment]
Same issue here.

**Comment by dvthust** [comment]
Please fix this issue on laravel 7 !

**Comment by tvbeek** [comment]
The problem is not directly part of this library. The download command download the zip from http://download.cksource.com/CKFinder/CKFinder%20for%20PHP/3.5.1/ckfinder_php_3.5.1.zip

Version 3.5.1 is the latest CKFinder version but the connector in that version conflicts with Laravel 7 because it use older Symfony packages. (So in fact it conflicts with with the newer Symfony components.)

The problem is a problem in the CKFinder connector.

**Comment by sunardo** [comment]
@tvbeek do you know hot to fix or at least temporary solution?

**Comment by zaak** [comment]
As @tvbeek pointed out the issue is caused by outdated versions of Symfony components. We are working on this and soon we will release a new version that resolves this issue.

**Comment by MarekGogol** [comment]
Same issue.

**Comment by richienabuk** [comment]
@zaak Please, Is there any update on this?
How soon should we expect the fix?

Thanks for your effort.

**Comment by zaak** [comment]
Hi @richienabuk. The new version should be released next week.

**Comment by zaak** [comment]
Support for Laravel 7 has been added in just released `v3.5.1.4`.

**Comment by congngokmt** [comment]
Thank @zaak
